src/dawn/pipeline_info.h

```diff
@@ -53,9 +64,16 @@ struct RenderPipelineInfo {
   ::dawn::Buffer fb_buffer;
   std::vector<::dawn::Buffer> vertex_buffers;
   ::dawn::Buffer index_buffer;
+  /// storage and uniform buffers
+  std::vector<::dawn::Buffer> buffers_;
```

Comment:
Nit:  *class* member names end with an underscore, but *struct* member names do not.